General Fix #2: 2-23-18

Nashatok opened the pull request:
General fixes directed towards the fighters, including but not limited to:
-Rebalanced maneuverability for most fighters
-fixed/adjusted Weapons
-Added/removed weapons as necessary, to comply better with lore
-General modernization: updated Allegiance for many fighters, added Torpedoes where necessary.

Nashatok committed:
General Fix 2-23-18

Spent a good long while fixing up and/or improving the ships that were already included, mostly the fighters, ranging from the TIE variants, to the Havoc pirate fighter.
Most fixes include:
-Readjusting weapons
-adding torpedoes to certain ships, within reason.
-buffing/fixing manuverability
-other misc. changes

Nashatok committed:
Caught bug with a few Capital Ships - fixed

Forgot that at some point I tried to simplify a few things, then opted out, but forgot to clean up after.

Fixed now, though~

Servius commented:
You should be able to just update a pull request with new commits. Shouldnt need to do close+open everytime you want to do more

Nashatok commented:
Ah, alright. I'll keep that in mind.
Either way, everything's working now.

Servius commented:
Any chance youd be willing to post an “Upcoming Changes” discussion on the eu addon detailing the changes coming?

Nashatok commented:
Yeah, I can do that.

Nashatok committed:
Update 2/26/18

-Continued updating Allegiances
-Finished 3 more Capital Ships:
CC-7700 Corvette (Rebel)
MC-30 Destroyer (Rebel)
Patrol Frigate (CIS)

Nashatok commented:
Alright, it should be all set for update.
